wasm/client.go

```diff
@@ -344,6 +344,7 @@ func Client_RecvFile(_ js.Value, args []js.Value) interface{} {
 
 		readerObj := NewFileStreamReader(ctx, msg)
 		readerObj.Set("cancel", js.FuncOf(func(_ js.Value, args []js.Value) interface{} {
+			msg.Reject()
```

Comment:
Is this cancel called also when transfer is already happening? If yes, then we will get an error because of this check: https://github.com/LeastAuthority/wormhole-william/blob/7c33f0c7d3d01c8ace73079c16506641da563672/wormhole/recv.go#L376

Reply:
Probably what will happen during the transfer, that we will close mailbox with Errory: https://github.com/LeastAuthority/wormhole-william/blob/7c33f0c7d3d01c8ace73079c16506641da563672/wormhole/recv.go#L203

Reply:
@donpui we don't call this cancel function in winden during transfer since it doesn't work currently. We still use the browser refresh workaround. Though I can rename `cancel` to `reject` to be more clear